### ISSUE TITLE
Improve the Gallery

### ISSUE THREAD

**Issue by karthiknadar1204** [Anupkjha2601/finalproj#13]

**Comment by karthiknadar1204** [comment]
I would like to work on this issue under gssoc'23

**Comment by adityavpawar** [comment]
Please assign me this issue so that I can resolve it under GSSOC-23

**Comment by Anupkjha2601** [comment]
how you improving gallery ? give some picture what you are changing .

**Comment by pjyoti121** [comment]
please assign me this issue

**Comment by UditSharma9999** [comment]
I'm a contributor for GSSoC '23, I want to work on this issue. Please assign me.

**Comment by karthiknadar1204** [comment]
Hii,so sorry for the late response,had my exams going on

**Comment by karthiknadar1204** [comment]
Hii,so sorry for the late response,had my exams going on
 

Ok so right now the images are placed on the screen, i would like to add them inside individual containers and save them  across the horizontal area, add border radius to the images and a box shadow to the container in which the images will be out.

On hovering,i would like to add some border radius to the button  healthy food along with some transform and scaling effects to the button.

What do you think of these changes @Anupkjha2601 .

Please share anything you would like

**Comment by PARTH-1712** [comment]
I'm a contributor for GSSoC '23, I want to work on this issue. Please assign me.

**Comment by KamalSritha-04** [comment]
Can you please assign me this issue under GSSOC'23?

**Comment by karthiknadar1204** [comment]
@Anupkjha2601 should i start working?

**Comment by Anupkjha2601** [comment]
Yes @karthiknadar1204, you can do it. I assigned it to you. Sorry for not replied earlier.

**Comment by karthiknadar1204** [comment]
Thank you,will get to working on it @Anupkjha2601

**Comment by karthiknadar1204** [comment]
@Anupkjha2601 I just started the project on my local system, the hosted version looks hugely different from the one on my local server, and I can't figure out the reason.
The original site/hosted version:
<img width="1776" alt="Screenshot 2023-06-18 at 4 42 01 PM" src="https://github.com/Anupkjha2601/food-recipes-website/assets/95973644/461bb7ac-b4ec-4868-9ea2-d4f30648b4e0">

Local version:

<img width="1768" alt="Screenshot 2023-06-18 at 4 42 12 PM" src="https://github.com/Anupkjha2601/food-recipes-website/assets/95973644/373fb4e2-a334-4209-a8c9-a27f68ed98ea">

**Comment by karthiknadar1204** [comment]
Both of them are of the home page, is the project under development from the scratch or is there some issue on my side @Anupkjha2601

**Comment by Anupkjha2601** [comment]
No , the website get some fault and I and mentor working on it to remove this issue as soon as possible. I think there is some error in someone style.css in merged pull request so working to correct it.

**Comment by karthiknadar1204** [comment]
ohk, so will resume working on this issue once the bug is fixed, dose that work @Anupkjha2601

**Comment by Anupkjha2601** [comment]
Yes, sorry for that.

**Comment by karthiknadar1204** [comment]
@Anupkjha2601 Is the issue fixed?
